test/wasm_test/jest/global.setup.js

```diff
@@ -0,0 +1,5 @@
+import { config } from "dotenv";
+
+export default async () => {
+  config();
```

Comment:
Note that `config` by default pulls `.env`. If it fails, it doesn't explode. It returns an object of which contains an error. The upstream pipelines should remain unaffected for the definition of `AMALGAM_WASM_DIR`.